src/clr-angular/data/datagrid/datagrid-column-toggle-button.ts

```diff
@@ -39,5 +46,6 @@ export class ClrDatagridColumnToggleButton {
         changes: [DatagridColumnChanges.HIDDEN],
       })
     );
+    this.allSelected.next(true);
```

Comment:
We're missing the case where you select them all individually, I'm not sure if we want to announce at this point too or not?

Reply:
It was about getting response. If the user selects one by one, he gets "option x, checked" feedback. With the Select All it was just silence.